src/schema/objects/entities.yaml

```diff
@@ -422,3 +422,22 @@ tracksys:
     may be longer and more human readable.
   type: string
   format: label
+stimulus:
+  name: stim
+  display_name: Stimulus
+  description: |
+    The `stim-<label>` entity can be used to distinguish stimulus files stored in the `stimuli/` directory.
+
+    This entity is only applicable to stimulus files.
+  type: string
+  format: label
+annotation:
+  name: annot
```

Comment:
```suggestion
  name: ann
```

<details>
<summary>although we do have some IMHO obnoxious length 9 "recording", most of the entities short names are 2-4 characters long</summary> 

```shell
❯ awk '/^ *name:/{print length($2) " " $2;}' src/schema/objects/entities.yaml | sort -n | nl
     1	2 ce
     2	2 mt
     3	3 acq
     4	3 den
     5	3 dir
     6	3 inv
     7	3 mod
     8	3 rec
     9	3 res
    10	3 run
    11	3 seg
    12	3 ses
    13	3 sub
    14	3 trc
    15	4 desc
    16	4 echo
    17	4 flip
    18	4 hemi
    19	4 part
    20	4 proc
    21	4 task
    22	5 chunk
    23	5 label
    24	5 space
    25	5 split
    26	5 stain
    27	6 sample
    28	8 tracksys
    29	9 recording
```
</details>

so I wonder if `ann` is enough descriptive/non-ambigous (would not collide with some possible sensible future entity)?  According to chatgpt here is the most popular words starting with `ann` (includes "annotation"):

Anniversary
Announcement
Annulment
Annotation
Annex
Annihilation
Announcer

so I think it would be safe to assume we would not collide

Reply:
probably scrape it -- `annot` is much more descriptive!